templates/user/index.html.twig

```diff
@@ -1,16 +1,16 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Espace utilisateur{% endblock %}
+{% block title %}{{ 'nav.section.volunteer' | trans }}{% endblock %}
 
 {% block body %}
     {{ include('misc/flash-messages.html.twig') }}
-    <h1>Bienvenue, {{ app.user.fullName }}</h1>
-    <p>NIVOL : {{ app.user.identificationNumber }}</p>
-    <p><a class="btn btn-primary" href="{{ path('user_edit') }}" role="button">Modifier mes informations</a></p>
+    <h1>{{ 'nav.welcome' | trans }}, {{ app.user.fullName }}</h1>
```

Comment:
here you should use a parameter with a `%name%` parameter, as we won't always have `word` + `,` + `name`.

For example : `nav.welcome: Hello %name%`.

`{{ 'nav.welcome'|trans({'%name%': app.user.fullName)) }}`

Reply:
https://symfony.com/doc/current/translation/templates.html